lang/en/texts/update-a-product-from-the-mobile-app.html

```diff
@@ -0,0 +1,666 @@
+		</figure>
+	</div>
+
+	<div class="small-12 medium-3 columns">
+		<figure>
+		    <img src="/images/misc/mobile-app-help/android-fr-nutrition-cropped.png"
+		         alt="">
+		    <figcaption>You can use the 4 white corners to resize the bounding box.  if you have trouble moving the corners due to the edge of the screen, you can use the rotate button adjust the corner and then rotate back.</figcaption>
+		</figure>
+	</div>
+
+	<div class="small-12 medium-3 columns">
+		<figure>
+		    <img src="/images/misc/mobile-app-help/android-fr-cropped-nutrition-results.png"
+		         alt="">
+		    <figcaption>Congratulations, you have adjusted the nutrition picture.</figcaption>
+		</figure>
+	</div>
+
+</div>
+
+<div class="row">
+	<div class="small-12 medium-6 columns">
+		<figure>
+		    <img src="/images/misc/mobile-app-help/android-fr-cropped-nutrition.png"
+		         alt="">
+		    <figcaption></figcaption>
+		</figure>
+	</div>
+
+
+	<div class="small-12 medium-6 columns">
+		<figure>
+		    <img src="/images/misc/mobile-app-help/android-fr-select-image-confirm.png"
+		         alt="">
+		    <figcaption></figcaption>
+		</figure>
+	</div>
+	<div class="small-12 medium-6 columns">
+		<figure>
+		    <img src="/images/misc/mobile-app-help/android-fr-ingredients-cropped.png"
+		         alt="">
+		    <figcaption></figcaption>
+		</figure>
+	</div>
+
+</div>
+
+<div class="row">
+
+	<div class="small-12 medium-6 columns">
+		<figure>
+		    <img src="/images/misc/mobile-app-help/android-fr-ingredients-result.png"
+		         alt="">
+		    <figcaption>Congratulations, you have adjusted the ingredient picture.</figcaption>
+		</figure>
+	</div>
+
+	<div class="small-12 medium-6 columns">
+		<figure>
+		    <img src="/images/misc/mobile-app-help/android-fr-ingredients-results.png"
+		         alt="">
+		    <figcaption>The cropped version of the front photo now constitutes a very good ingredient photo.</figcaption>
+		</figure>
+	</div>
+
+	<div class="small-12 medium-6 columns">
+		<figure>
+		    <img src="/images/misc/mobile-app-help/android-fr-uploading.png"
+		         alt="">
+		    <figcaption>The cropped instruction for the nutrition table are being sent to the server.</figcaption>
+		</figure>
+	</div>
+</div>
+
+<!-- Take a photo -->
+<h3 style="background-color:#09f;padding:1%;color:#fff;font-weight:bold;">1. Take missing photos</h3>
+<div class="row">
+<div class="small-12 medium-6 columns">
+	<figure>
+	    <img src="/images/misc/mobile-app-help/android-fr-take-a-photo.png"
+	         alt="">
+	    <figcaption>Due to the collaborative nature of open food facts,  some photos may be missing.  just click on the take photo icon to take part in food transparency.</figcaption>
+	</figure>
+</div>
+</div>
+
+<p class="callout panel">You need the Android Open Food Facts app to update a product that way. You can get it <a href="https://android.openfoodfacts.org/">there</a>.</figcaption>
```

Comment:
What about the website or iOS app? You don't *need* the Android app …

Reply:
this tutorial is Android only at this point. We don't have such advanced features in the iOS app